multicast/exceptions.py

```
@@ -331,15 +333,15 @@ class ShutdownCommandReceived(RuntimeError):
 			'Custom shutdown message.'
 	"""
 
-	__module__ = """multicast.exceptions"""
+	__module__ = "multicast.exceptions"
 
-	__name__ = """multicast.exceptions.ShutdownCommandReceived"""
+	__name__ = "multicast.exceptions.ShutdownCommandReceived"
 
-	def __init__(self, *args, **kwargs):
+	def __init__(self, *args, **kwargs) -> None:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)